twilio.rb

```diff
@@ -0,0 +1,35 @@
+# # require 'rubygems' # not necessary with ruby 1.9 but included for completeness
```

Comment:
Metrics/LineLength: Line is too long. [82/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)